axelarnetwork/axelar-core
# feat(cmd): add command to set genesis gov params

## Description
This pr add command to set following gensis gov params
- minium deposit coin
- max deposit period
- voting period

## Todos


## Steps to Test
Add cmd in create genesis script
```
axelard set-genesis-gov --minimum-deposit 512000000uaxl --max-deposit-period 3600000000000 --voting-period 3600000000000
```
Use `axelard q gov params` to check the params
## Expected Behaviour

## Other Notes
